01LessonOne/LessonOne.js

```diff
@@ -31,7 +41,11 @@ module.exports = function() {
 
   this.notEqualTo = function (a, b) {
     // should return true if the first argument is not equal to the second argument
-    // and should return false if the first argument is equal to the second argument
-
+    if (a !== b){
```

Comment:
Compare this to the solution provided for `this.equalTo`. What's different between this solution and the `this.equalTo` solution? Is the difference necessary?